api/SubmitJob/src/main.py

```diff
@@ -0,0 +1,27 @@
+import json
+from os import getenv
```

Comment:
I think `environ` is preferable, we'll get a specfic exception when referencing a variable that doesn't exist, rather than `submit_job` complaining about an invalid `None` parameter